Makefile

```diff
@@ -693,6 +698,10 @@ ifeq (${DYN_DISABLE_AUTH},1)
 $(eval $(call add_define,DYN_DISABLE_AUTH))
 endif
 
+ifneq ($(findstring armlink,$(notdir $(LD))),)
+$(eval $(call add_define,BL31_USE_ARMLINK))
```

Comment:
Sorry, didn't notice this previously. Could you rename `BL31_USE_ARMLINK` to just `USE_ARM_LINK` and then add userguide instruction for the build option ?

Reply:
Thats ok. I pushed a new update